src/machines/billingMachine.ts

```diff
@@ -109,6 +117,12 @@ export const billingMachine = setup({
             target: BillingState.Waiting,
             // Yep, this is hard to follow. XState, why!
             actions: assign({
+              // Clear out the rest of the fields here
+              credits: undefined,
+              allowance: undefined,
+              isOrg: undefined,
+              hasSubscription: undefined,
+              lastFetch: new Date(),
```

Comment:
This was found while debugging: I don't believe we should keep the context from prior calls on error, and instead should reset them to defaults